tests/unit/django_spanner/simple_test.py

```diff
@@ -7,13 +7,16 @@
 from django_spanner.client import DatabaseClient
 from django_spanner.base import DatabaseWrapper
 from django_spanner.operations import DatabaseOperations
-from unittest import TestCase
+
+# from unittest import TestCase
+from tests._helpers import OpenTelemetryBase
 import os
 
 
-class SpannerSimpleTestClass(TestCase):
+class SpannerSimpleTestClass(OpenTelemetryBase):
```

Comment:
It seems like most test classes won't also test tracing, any reason not to make a separate test class for tracing? E.g. `SpannerTracingTestClass`?

And what about making the django tests use [django's `TestCase`](https://docs.djangoproject.com/en/2.2/topics/testing/overview/) instead of `unittest.TestCase`?

Something like:

```python
from django.test import TestCase

class SpannerSimpleTestCase(TestCase)
    ...

class SpannerTracingCase(SpannerSimpleTestCase, OpenTelemetryBase)
    ...
```

Reply:
the [django TestCase](https://docs.djangoproject.com/en/2.2/topics/testing/overview/) inherits from `TransactionTestCase` which inherits from `SimpleTestCase` which does database connection check, and we don't connect to db for our unit tests so it's simpler to not use it at least for unit tests.